wave_utils/include/wave/utils/math.hpp

```diff
@@ -37,6 +37,9 @@ void mat2vec(MatX A, std::vector<double> &x);
 double wrapTo180(double euler_angle);
 double wrapTo360(double euler_angle);
 int euler2rot(Vec3 euler, int euler_seq, Mat3 &R);
+int euler2quat(Vec3 euler, int euler_seq, Quaternion &q);
```

Comment:
OK, in that case I would suggest changing these methods. From [my previous comment](https://github.com/wavelab/libwave/pull/24#discussion_r107806370) on rotation PR: 

Use Eigen's methods e.g. http://stackoverflow.com/questions/21412169/creating-a-rotation-matrix-with-pitch-yaw-roll-using-eigen

Perhaps both the Rotation class and this module should use the same methods to avoid duplication.